### PR TITLE
Additional DATAFORMAT guidance per previous discussions.

### PR DESCRIPTION
**Pull request by jasonwr**
…strative example of graph (traffic) data as opposed to the existing one in DATAFORMAT.md Will later submit as a PR to the main repo.

### CODE REVIEW

File: DATAFORMATS.md [hunk 1]
```diff
@@ -58,6 +58,265 @@ The traffic data expected to be passed in regularly to `ele.updateData(traffic)`
...
+        {
+          "name": "foxtrot",
+          "metadata": {
+            "streaming": true
+          },
+          "class": "normal"
+        }
+      ],
+      "connections": [
+        {
```
Comment (jrsquared) [inline DATAFORMATS.md]: Remove connections that have no target. I notice that this was just copied from [vizceral-example/src/sample_data_simple.json](https://github.com/Netflix/vizceral-example/blob/master/src/sample_data_simple.json) so I removed those dead connections from there as well. I also removed some extraneous data.

**Comment by jrsquared** [comment]
Example usage in vanilla HTML/JS and example data don't belong in DATAFORMATS.md. I could see adding the standard HTML as a non-ES6 example in the README.md right after the existing example in step 2 of [Using](https://github.com/Netflix/vizceral#using).  I would also take out all of the XHR code and just load the inline simple data structure that is in the README to keep the example as simple and short as possible.

**Comment by jasonwr** [comment]
@jrsquared sorry I haven't responded. I've been busy with grad school (CS) and work. I'll try to get on tonight and work on the README.md When you say:  "just load the inline simple data structure that is in the README" That example is so basic that I'm concerned developers won't find interest in Vizceral by just looking at this. The one-to-many example I provided really isn't that complex and illustrates the region level view. I know for the stuff I'm working on is all region based, and I don't even have a global view.

**Comment by jrsquared** [comment]
[vizceral-example](https://github.com/netflix/vizceral-example) exists as a proof of concept showing what vizceral can do.  Using React like the example uses or not is just an implementation detail.

I do find it useful having a short and sweet example showing that it can be used without any framework of any kind, so I am happy to add that to the README.  It's easy enough for someone to copy and paste the json from vizceral-example that I don't want to start to put more complex example data into this repo at this time.

I updated the README.md in 99f7d228c1d27529d7f2a0c9747288db34b8cefc to include the vanilla example.

Thank you for your help in pushing to make sure it worked in regular JS, much appreciated.

**Pull request closed without merging**